gitguard.py

```diff
@@ -249,3 +254,128 @@ def get_commit_history(repo_link, author_name=None, start=None, end=None, path=N
         commit['commit_message'] = commit_history[i]["commit"]["message"]
         history.append(commit)
     return history
+
+def _remove_readonly(func, path, excinfo):
+    """
+    Helper function to remove change read-only file status to editable
+    Source: http://stackoverflow.com/questions/1889597/deleting-directory-in-python
+    """
+    os.chmod(path, stat.S_IWRITE)
```

Comment:
the runner(user) might not have access right to chmod.
Might want to consider try except or something.
I'm okay with ignoring too.